content/pt-br/docs/concepts/policy/node-resource-managers.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gerenciadores de Recursos de Nós
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Para dar suporte a latência crítica e a cargas de trabalho de alta taxa de transferência, o Kubernets oferece um conjunto de gerenciadores de recursos. Os gerenciadores visam coordenar e otimizar o alinhamento de recursos do nó(s) para Pods configurados com um requisito específico para CPUs, dispositivos e recursos de memória (*hugepages*).
```

Comment:
Sounds better to me: "Para dar suporte a cargas de trabalho com latência crítica e altas taxa de transferência...". Think about it.

Minor: "Kubernetes" instead of "Kubernets"

Reply:
It really makes more sense, I'll change